test_unstructured_inference/models/test_detectron2.py

```diff
@@ -10,14 +10,17 @@ def __init__(self, *args, **kwargs):
         self.args = args
         self.kwargs = kwargs
 
+    def detect(self, x):
+        return "called"
```

Comment:
Is this supposed to return a string? Totally fine if so, just felt a little odd.

Reply:
I don't do anything with the output. Maybe it's a bit misleading this way though.

Reply:
Yeah I'd probably return `None` if you don't do anything with it, but I don't feel too strongly about it

Reply:
Changed output type to `MockLayout`